notebooks/harmony_concatenation/Harmony_Concatenation.ipynb

```diff
@@ -2,839 +2,294 @@
  "cells": [
```

Comment:
<p>The formatting on this text looks a bit weird. Since this version is end-user focused let's actually remove the reference to UAT and just talk about OPS environment now. I would also leave the original text about the .netrc file</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/podaac/tutorials/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>